tasks-ui.md

```diff
@@ -0,0 +1,27 @@
+# Tasks
+
+ - [ ] Clean up App.js (5 mins)
+ - [ ] Add styling to Storage Summary (5-10 mins)
+ - [ ] Dry up StorageSummary component (10 mins)
+ - [ ] Issue PR (you can use `git publish` on this machine)
```

Comment:
Maybe I should remove the part in parentheses here, since it is specific to a git alias on my machine?

Reply:
I'm going to take care of this

Reply:
Done👍